src/components/Products/ProductDetails.js

```diff
@@ -0,0 +1,81 @@
+import React, { useState, useEffect, useContext } from "react";
+import "./ProductDetails.css";
+import { useParams } from "react-router-dom";
+import { useHistory } from "react-router-dom";
+import { CartContext } from "../../reducer/Context";
+
+function ProductDetails() {
+  const [productDetails, setProductDetails] = useState({});
+  const [qty, setQty] = useState(1);
```

Comment:
its often helpful to have more readable var names like quantity/count setQuantity etc. Makes code more readable. Just a suggestion.